common/djangoapps/xmodule_modifiers.py

```diff
@@ -277,6 +277,7 @@ def add_staff_markup(user, has_instructor_access, block, view, frag, context):
                      'category': str(block.__class__.__name__),
                      # Template uses element_id in js function names, so can't allow dashes
                      'element_id': block.location.html_id().replace('-', '_'),
+                     'region': unicode(course_id).replace(':', '').replace('/', '_').replace('+', '_'),
```

Comment:
@cpennington there is no active region here we need this for sudo modal to submit user password to get sudo access.
I will create a function to map ```CourseKey``` to region.